internal/apiserver/server.go

```diff
@@ -476,9 +476,15 @@ func (as *apiServer) createMuxRouter(ctx context.Context, o orchestrator.Orchest
 				Methods(route.Method)
 		}
 	}
+	r.HandleFunc("/api/v1/namespaces/{ns}/apis/{apiName}/ui", func(rw http.ResponseWriter, req *http.Request) {
+		path := strings.TrimSuffix(req.URL.Path, "/ui") + "/apispec"
+		handler := as.apiWrapper(as.swaggerUIHandler(publicURL + path))
+		handler(rw, req)
```

Comment:
Is there any cleaner way to build this handler?

Reply:
Think this is fine - this isn't a performance sensitive path.